frontend/src/components/editor/Cell.tsx

```diff
@@ -654,7 +655,10 @@ const EditableCellComponent = ({
                 )}
               </div>
             </div>
-            <SqlValidationErrorBanner cellId={cellId} />
+            <SqlValidationErrorBanner
+              cellId={cellId}
+              hide={cellRuntime.errored}
```

Comment:
if it errored and you are trying to fix it, this won't show then? 
should we do `hide={cellRuntime.errored && !isStaleCell}`?